pyscraper/sp/get-official-reports-new.py

```diff
@@ -92,8 +109,13 @@ def main():
         for i in range(1, int(options.historic, 10)):
             get_document_from_id(i)
 
+    elif options.start_range and options.end_range:
+        for i in range(int(options.start_range), int(options.end_range, 10)):
```

Comment:
`range()` doesn't include the end number, so I think yo want a +1 here to cover the end number asked for.